### PR TITLE
KevS-SK1544-KFT0016205-Update-Htaccess-with-WordPress-for-WebHosting

### PR DESCRIPTION
**Pull request by Kazarus4**
FRFR updates done.

Need peer rv by @OVH-Team-Guides for FRFR subs before FRCA duplications and translations for the other subs.

After that, i'll need a tech rv to be sure that all is ok.

### CODE REVIEW

File: pages/web_cloud/web_hosting/htaccess_how_to_protect_wordpress/guide.fr-fr.md [hunk 1]
```diff
@@ -136,28 +136,60 @@ Votre fichier **wp-config.php**, présent à la racine de votre site web, contie
...
+>>
+>>```bash
+>> <Limit GET POST PUT>
+>>   order deny,allow deny from all
+>>   allow from 203.0.113.0
+>>   allow from 203.0.113.1
+>> </Limit>
+>>```
+>>
+>> Dans cet exemple, seuls les adresses IPv4 `203.0.113.0` et `203.0.113.1` sont autorisées à accéder au répertoire dans lequel le fichier .htaccess se trouve.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/htaccess_how_to_protect_wordpress/guide.fr-fr.md]:
```suggestion
>> Dans cet exemple, seules les adresses IPv4 `203.0.113.0` et `203.0.113.1` sont autorisées à accéder au répertoire dans lequel se trouve le fichier .htaccess.
```

File: pages/web_cloud/web_hosting/htaccess_how_to_protect_wordpress/guide.fr-fr.md [hunk 2]
```diff
@@ -136,28 +136,60 @@ Votre fichier **wp-config.php**, présent à la racine de votre site web, contie
...
+>>
+>>```bash
+>> <Limit GET POST PUT>
+>>   order deny,allow deny from all
+>>   allow from 2001:db8:1:1b00:203:0:113:0
+>>   allow from 2001:db8:1:1b00:203:0:113:1
+>> </Limit>
+>>```
+>>
+>> Dans cet exemple, seuls les adresses IPv6 `2001:db8:1:1b00:203:0:113:0` et `2001:db8:1:1b00:203:0:113:1` sont autorisées à accéder au répertoire dans lequel le fichier .htaccess se trouve.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/htaccess_how_to_protect_wordpress/guide.fr-fr.md]:
```suggestion
>> Dans cet exemple, seules les adresses IPv6 `2001:db8:1:1b00:203:0:113:0` et `2001:db8:1:1b00:203:0:113:1` sont autorisées à accéder au répertoire dans lequel se trouve le fichier .htaccess.
```

**Comment by Kazarus4** [comment]
Tech rv started

**Comment by Kazarus4** [comment]
Scripts corrected

**Pull request merged**